src/clientManager.js

```diff
@@ -32,12 +32,29 @@ ClientManager.prototype.findClient = function(name) {
 };
 
 ClientManager.prototype.autoloadUsers = function() {
-	this.getUsers().forEach((name) => this.loadUser(name));
+	const users = this.getUsers();
+	const noUsersWarning = `There are currently no users. Create one with ${colors.bold("lounge add <name>")}.`;
+
+	// There was an error, already logged, but we have to crash the server as
+	// user directory could not be accessed
+	if (users === undefined) {
+		process.exit(1);
+	}
+
+	if (!users.length) {
+		log.warn(noUsersWarning);
+	}
+
+	users.forEach((name) => this.loadUser(name));
 
 	fs.watch(Helper.USERS_PATH, _.debounce(() => {
 		const loaded = this.clients.map((c) => c.name);
 		const updatedUsers = this.getUsers();
 
+		if (!updatedUsers.length) {
```

Comment:
I figured it couldn't hurt. This would warn if something happened that led The Lounge to not have any users anymore, which is pretty useful when dealing with FS snafus. In particular, it logs the time this happened, which could be very useful.

I totally get what you're saying, but it's free for us to warn there (including code maintenance) and it adds some value in case something goes wrong, so I think we can afford leaving it here. What do you think? :)